extensions/web-tracker/wab/src/main/resources/META-INF/cxs/schemas/articleCompleted.json

```diff
@@ -2,23 +2,18 @@
   "$id": "https://unomi.apache.org/schemas/json/events/articleCompleted/1-0-0",
   "$schema": "https://json-schema.org/draft/2019-09/schema",
   "self":{
-    "vendor":"org.apache.unomi",
+    "vendor":"The Apache Software Foundation",
```

Comment:
Careful, this schema doesn't contains "unevaluatedProperty:false", so it could produce mapping explosion